.github/workflows/publish-docs.yaml

```diff
@@ -32,6 +38,7 @@ jobs:
     - name: Build stable (i.e. from `release`) documentation
       working-directory: docs
       run: |
+        echo "Building documentation for stable version: $DOCS_VERSION"
```

Comment:
Where is this env var being set?

Reply:
You are right, I was supposed to get it from the `get_version` step. Fixed that now!